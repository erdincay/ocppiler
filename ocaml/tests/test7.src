let y = 5 in (y-1)
