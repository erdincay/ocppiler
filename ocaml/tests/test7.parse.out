(let y : int = 5 in (y + 1))
